CZI.md

```diff
@@ -107,73 +82,64 @@ the graduate student who is leading the this effort as her thesis work.
-from fixing bugs, reviewing Pull Requests, triaging bug reports,
-tagging release, to keeping the continuous integration services
-running.
+proposed work. Maintenance covers a wide range of tasks including 
+triaging bug reports, fixing some of those bugs, reviewing Pull Requests, 
+tagging releases, and keeping the continuous integration services
+running
 
 These task are essential for the project's health and each
-individually are small, but are best handled by a paid developer.
-These tasks are frequently reactive, time critical, and not always
-"fun" which is awkward to rely on solely volunteer effort to
-accomplish in a timely manner.  In addition to the inherent value of
-completing these tasks, having them promptly and reliably addressed
-improves the contribution experience for everyone working on the
-project.  We propose to devote three quarters of a developers time to
-handling these tasks.
+individually are small, but are frequently reactive, time critical, and 
+often tedious. They are best handled by a paid developer because it is 
+awkward to rely on solely volunteer effort to accomplish tasks that they often 
+do not have intrinsic interest in carrying out. 
+
+In addition to the inherent value of completing these tasks, having them 
+promptly and reliably addressed improves the contribution experience for 
+everyone working on the project.  We propose to devote three quarters of a 
+developers time to handling these tasks.
 
 In addition to the reactive work there are substantial but incremental
 improvements that will benefit from long blocks of dedicated work.
 Examples of this work include fixing long standing rendering and
 performance issues, deep-dive documentation, homogenizing and
 smoothing the API, and new user-facing functionality. For example, we
-have a long-standing prototype adding a color-blind simulation filter
-to the UI.  To turn this prototype into a user-ready functionality we
+have a long-standing prototype for adding a color-blind simulation filter
+to the UI.  To turn this prototype into user-ready functionality, we
 need to do additional research to make sure we are implementing the
 filters accurately in addition to the engineering effort.  We propose
 to devote about a quarter of a developer to this effort which should
-enable us to complete 5 of these projects.  We will work with
-downstream bio-projects to identify what will have the greatest
-impact.
-
-As a library Matplotlib needs to be able to support the use cases for
-the full range of science domains and complex novel visualization [1]
-that provide new insight.  At the same time the most common
-visualizations in a domain need to be fluid for the end-practitioners,
-with the "obvious" customization options exposed.  To achieve both of
-these goals we need to continue to foster a two layer eco-system with
+enable us to complete 5 of these projects.  
+<!--confused about 5 'cause you only mentioned this color blind thing that feels random-->
```

Comment:
5 projects in ~2.5 weeks FTE per project which sets an expectation for the scale of these projects.  

These are small enough that I don't think we want to try and identify them all right now (as some of them might get done in the next 6mo) and if we come across a really good idea in the next 18mo we don't want to write it out of scope.

Reply:
>5 projects in ~2.5 weeks FTE per project which sets an expectation for the scale of these projects.
:/ eh...I don't really think so, as that's dependent on the FTE skillset. But then something like:
Funding would also enable us to tackle about 5 medium sized fairly self contained improvements to the project, such as new features and performance optimizations, that we expect to each take about 3 weeks of effort. Without funding, at best these sorts of projects often drag out for months and sometimes stall out altogether.

Reply:
I like that wording a lot better.